src/experimental-features.js

```diff
@@ -0,0 +1,26 @@
+export class ExperimentalFeatures {
+  constructor(storage) {
+    this.storage = storage;
+  }
+  getExperimentalFeatures() {
+    return this.storage.getItem('dopplerExpermiental')
+      ? JSON.parse(this.storage.getItem('dopplerExpermiental'))
+      : null;
+  }
+  isFeatureEnabled(feature) {
+    const experimentalFeatures = this.getExperimentalFeatures();
+    if (experimentalFeatures) {
+      return !!experimentalFeatures[feature];
```

Comment:
i not sure but maybe can replace with `return experimentalFeatures && !!experimentalFeatures[feature];` and dont need return false